src/lib/autoinstall/widgets/storage/partition_table.rb

```diff
@@ -40,19 +40,22 @@ def label
           _("Partition table")
         end
 
-        ITEMS = [:msdos, :gpt, :none].freeze
-        ITEMS_LABELS = {
-          msdos: N_("MSDOS"),
-          gpt:   N_("GPT"),
-          none:  N_("None")
-        }.freeze
-        private_constant :ITEMS, :ITEMS_LABELS
+        TYPES = [
+          Y2Storage::PartitionTables::Type::GPT,
+          Y2Storage::PartitionTables::Type::MSDOS,
+          Y2Storage::PartitionTables::Type::DASD
```

Comment:
do we intentionally skip some type? if so, please document it. Otherwise I would use simple `Y2Storage::PartitionTables::Type.all` or is also ordering important and if so, also please document.
So in short please use all or document why it cannot be used.